pyproject.toml

```diff
@@ -11,3 +11,25 @@ line_length = 100
 force_sort_within_sections = true
 # Combines "as" imports on the same line
 combine_as_imports = true
+
+[tool.mypy]
+ignore_missing_imports = true
+show_error_codes = true
+disable_error_code = ["attr-defined", "var-annotated"]
+follow_imports = "skip" # Don't follow imports into other files. This should be removed once all type errors have been resolved.
+exclude = [
+    "^dandiapi/api/admin.py",
+    "^dandiapi/api/mail.py",
+    "^dandiapi/api/tests/",
+    "^dandiapi/api/management/",
+    "^dandiapi/api/migrations/",
+    "^dandiapi/api/models/",
+    "^dandiapi/api/tasks/",
+    "^dandiapi/api/views/",
+]
+
+# Re-enable these when https://github.com/typeddjango/django-stubs/issues/417 is fixed.
+# plugins = ["mypy_django_plugin.main", "mypy_drf_plugin.main"]
+
+# [tool.django-stubs]
+# django_settings_module = "dandiapi.settings"
```

Comment:
Since we are using `django-configurations` (well, technically `composed-configuration` does, not dandi), there's some additional considerations for running these django plugins. There's an open issue to support it https://github.com/typeddjango/django-stubs/issues/417, but it hasn't been implemented yet

Reply:
DANDI _does_ use `django-configuration` directly. That's why things like this: https://github.com/dandi/dandi-archive/blob/master/manage.py#L14 are necessary.

I agree that due to the lack of proper support, it's safer to disable these plugins for now.